rules/linux/execution_expect_binary.toml

```diff
@@ -1,15 +1,15 @@
 [metadata]
 creation_date = "2022/03/07"
 maturity = "production"
-updated_date = "2022/03/31"
+updated_date = "2022/04/29"
 
 [rule]
 author = ["Elastic"]
 description = """
-Identifies Linux binary expect abuse to break out from restricted environments by spawning an interactive system shell.
-The expect utility allows us to automate control of interactive applications such as telnet,ftp,ssh and others and the
-activity of spawning shell is not a standard use of this binary for a user or system administrator and could potentially
-indicate malicious actor attempting to improve the capabilities or stability of their access.
+Identifies Linux binary expect command abuse to break out from restricted environments by spawning an interactive system
+shell. The expect utility allows us to automate control of interactive applications such as Telnet, FTP, SSH and others.
+The activity of spawning shell is not a standard use of this binary for a user or system administrator and could
```

Comment:
"The activity of spawning a shell is not a standard use of this binary by a user or system administrator. It indicates a potentially malicious actor attempting to improve the capabilities or stability of their access."

-This is the language I see being used in the other rules. I'm assuming you meant to change it here as well.

Reply:
Not sure I follow? That language is included here from what I see.